cluster/userdata-master.yaml

```diff
@@ -1063,6 +1063,17 @@ write_files:
       parameters:
         type: gp2
 
+  - path: /srv/kubernetes/manifests/thirdpartyresources/platform-i-a-m-credentials.yaml
+    content: |
+      apiVersion: extensions/v1beta1
+      kind: ThirdPartyResource
+      metadata:
+        # NOTE: dashes are needed to get proper PlatformIAMCredentials name
+        name: platform-i-a-m-credentials.zalando.org
```

Comment:
BTW: we might want to change it to a singular form like e.g. `PlatformIAMCredentialSet` as Kubernetes pluralizes the resource name in the HTTP path (leading to constructs like `platformiamcredentialses`).